src/templates/script/ScriptBundleTemplateProvider.ts

```diff
@@ -41,9 +38,7 @@ export class ScriptBundleTemplateProvider extends ScriptTemplateProvider {
         const resourcesUrl: string = release.resources.replace('{locale}', language);
 
         const urls: string[] = [release.bindings, resourcesUrl, release.functions];
-        const responses: HttpOperationResponse[] = await Promise.all(urls.map(url => requestUtils.sendRequestWithTimeout({ method: 'GET', url })));
-        // NOTE: r.parsedBody doesn't work because these feeds sometimes return with a BOM char or incorrect content-type
-        [this._rawBindings, this._rawResources, this._rawTemplates] = <[object, object, object[]]>responses.map(r => parseJson(nonNullProp(r, 'bodyAsText')));
+        [this._rawBindings, this._rawResources, this._rawTemplates] = <[object, object, object[]]>await Promise.all(urls.map(url => feedUtils.getJsonFeed(url)));
```

Comment:
Do you not have to specific the generic object T when you call `getJsonFeed`? Like `getJsonFeed<[object, object, object[]]>`?

Reply:
Nope, not necessary. If anything, It would actually be `getJsonFeed<object>` (remember, it's inside a `map` function`). Typescript is smart enough to infer stuff like that.